demo-terraform/aws/serverless-app/dynamodb.tf

```diff
@@ -2,29 +2,37 @@
 
 ## Dynamodb Table
 resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure server side encryption (SSE) is enabled for AWS DynamoDB tables**
 
**Vulnerability**
AWS DynamoDb tables have server side encryption (SSE) disabled. This may lead to data leak.
 
**Remediation**
Server Side Encryption can be enabled for AWS DynamoDB tables. 
 In AWS Console - 
 1. Sign in to AWS Console and go to the AWS DynamoDB table.
 2. Select Tables in the navigation pane.
 3. Select the necessary table.
 4. Find Encryption Type under Table details in the Overview tab.
 5. Ensure Encryption is enabled.
 6. Click Save. 
 In Terraform - 
 Set 'server_side_encryption.enabled' to 'true'.

source: accurics-